test/e2e/_helpers.js

```diff
@@ -75,6 +112,69 @@ export async function createPersonFixture() {
   }
 }
 
+/**
+ * Get a random location
+ *
+ * @param {string} [locationType=police] - type of location
+ *
+ * @returns {string} - location id
+ */
+const getRandomLocation = (locationType = 'police') => {
```

Comment:
Maybe we shouldn't have a default? Feels like a safer solution would be to use `all` as the default.

Also, if a `type` is given, it might be better to use `referenceData.getLocationsByType()`. Will be faster to do the filter on the API.

Reply:
Default removed and calls `getLocationsByType` if `locationType` provided, 'getLocations` if not

Reply:
Looks like the default snuck back in

Reply:
More likely I hallucinated taking it out